src/compiler/parse/state/tag.ts

```diff
@@ -472,12 +473,26 @@ function read_attribute_value(parser: Parser) {
 				/(\/>|[\s"'=<>`])/
 	);
 
-	const value = read_sequence(parser, () => !!parser.match_regex(regex));
-
-	if (quote_mark) parser.index += 1;
+	let value: TemplateNode[] = [];
```

Comment:
Minor code style suggestion: I would move the `if(is_blank)` section right below the `is_blank` declaration to signal the reader that the code bails early at this point.

Reply:
Thank you for reviewing.
100% collect!👍
I changed this and commited.

Reply:
I meant moving the `if(is_blank)` up to the top, not moving the `const` down. They way it's written right now is a bug because you the parser eats the first quote mark before the check.

Reply:
Oops.
Sorry for less understanding.

It means like the below code right?
(This function should be return early if an attribute value is an empty string.)

```
function read_attribute_value(parser: Parser) {
	const is_blank = parser.eat("''") ? true : parser.eat('""') ? true : false;
	if (is_blank) {
		return [flush_blank_node(parser)];
	}

	const quote_mark = parser.eat("'") ? "'" : parser.eat('"') ? '"' : null;
	const regex = (
		quote_mark === "'" ? /'/ :
			quote_mark === '"' ? /"/ :
				/(\/>|[\s"'=<>`])/
	);

	const value = read_sequence(parser, () => !!parser.match_regex(regex));
	if (quote_mark) parser.index += 1;
	return value;
}
```

And if it is okay,
can I do force-push to my previous commit?

Reply:
Yes that's what I meant. If you force push or do a thousand single commits is irrelevant, the PR will be squashed into a single commit and then merged.

Reply:
Got it! Thank you!
And I committed it.